hangman/lib/hangman/impl.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Hangman.Impl do
+  def new_game() do
+    %Hangman.Game{%Hangman.Game{} | letters: Dictionary.random_word |> String.codepoints}
```

Comment:
## -2 idiomatic

This is a wee bit baroque. How about just

~~~ elixir
def new_game() do
  %Hangman.Game{ 
    letters: Dictionary.random_word |> String.codepoints 
  }
end
~~~